gui/types/projectsTypes.ts

```diff
@@ -17,6 +18,7 @@ export interface ProjectTypes {
   project_description: string;
   project_hash_id: string;
   project_url: string;
+  project_repository: string | undefined;
```

Comment:
in ts, both null and undefined are treated as two different distinct types and mostly we deal with null than undefined so add that as well

Reply:
It's against typescript standard, https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined